packages/circuit-ui/components/SideNavigation/SideNavigation.tsx

```diff
@@ -82,7 +79,7 @@ export function SideNavigation({
       closeButtonLabel={closeButtonLabel}
       primaryNavigationLabel={primaryNavigationLabel}
       onClose={onClose}
-      open={!prevOpen && isOpen}
```

Comment:
@sirineJ, do you remember why this took into account whether the mobile navigation was previously open?